src/test/fixtures/generator.mjs

```diff
@@ -0,0 +1,138 @@
+import { configFileV2 } from "./configuration-file.mjs";
+import { ropsten, rinkeby, did } from "./credential.mjs";
+import { existsSync, writeFileSync, mkdirSync } from "fs";
+import { fileURLToPath } from "url";
+import { join, dirname, parse } from "path";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const V2_ROPSTEN_DIRECTORY = join(__dirname, "_generated", "v2", "config", "ropsten", "sample-config-ropsten.json");
+const V2_AWS_ROPSTEN_DIRECTORY = join(
+  __dirname,
+  "_generated",
+  "v2",
+  "config",
+  "ropsten",
+  "sample-config-ropsten-aws.json"
+);
+const V2_RINKEBY_DIRECTORY = join(__dirname, "_generated", "v2", "config", "rinkeby", "sample-config-rinkeby.json");
+const V2_DID_DIRECTORY = join(__dirname, "_generated", "v2", "config", "did", "sample-config-did.json");
+
+const V3_ROPSTEN_DIRECTORY = join(__dirname, "_generated", "v3", "config", "ropsten", "sample-config-ropsten.json");
+const V3_RINKEBY_DIRECTORY = join(__dirname, "_generated", "v3", "config", "rinkeby", "sample-config-rinkeby.json");
+const V3_DID_DIRECTORY = join(__dirname, "_generated", "v3", "config", "did", "sample-config-did.json");
+
+function generateConfigFile() {
+  //Ropsten
+  const ropstenConfigFile = JSON.parse(JSON.stringify(configFileV2));
+  ropstenConfigFile.network = ropsten.network;
+  ropstenConfigFile.wallet = ropsten.wallet;
+  ropstenConfigFile.documentStorage.url = ropsten.documentStorageUrl;
```

Comment:
```suggestion
const { network, wallet, documentStorageUrl } = ropsten;
  const ropstenConfigFile = {
    ...configFileV2,
    network,
    wallet,
    documentStorageUrl,
  };
```

Reply:
i can't use the expand method, cause rinkeby and did both use same properties.

Reply:
refactor abit?

Reply:
oh refracted